src/main/java/in/org/projecteka/hdaf/link/link/Link.java

```diff
@@ -17,7 +18,11 @@
 import static in.org.projecteka.hdaf.link.link.Transformer.toHIPPatient;
 
 public class Link {
-
+    public static final ClientError HIP_NOT_FOUND = new ClientError(
+            HttpStatus.NOT_FOUND,
+            new ErrorRepresentation(new Error(
+                    ErrorCode.HIPNotFound,
+                    "HIP not found")));
```

Comment:
```suggestion
                    "HIP not found")));
```
Is this a message we are going to send to the Mobile? if yes, don't do this please, our consumer doesn't necessarily need to know what's down unless required.